app/json/enumerations.json

```diff
@@ -55,7 +55,7 @@
     },
     {
       "preparagraph": "You can extend the `Enumeration` by extending the `Val` class.",
```

Comment:
Good catch! We should change Val to Value in the preparagraph too, care to make the change? Thanks a lot.